keel-sqs/src/main/kotlin/com/netflix/spinnaker/config/SqsProperties.kt

```diff
@@ -5,4 +5,6 @@ import org.springframework.boot.context.properties.ConfigurationProperties
 @ConfigurationProperties("sqs")
 class SqsProperties {
   var queueARN: String = ""
+  var visibilityTimeoutSeconds: Int = 60
+  var waitTimeSeconds: Int = 10
```

Comment:
How long the `receiveMessages` call will block for (shorter if a message arrives). We're in a coroutine scope when we read from the queue so there's no point busy-waiting.

Reply:
I'm using 5 in echo

Reply:
But you're probably not using coroutines